app/plugin.go

```diff
@@ -460,6 +463,25 @@ func (a *App) DeletePublicKey(file string) *model.AppError {
 	return nil
 }
 
+// VerifyPlugin verifies plugin signature
```

Comment:
Could we elaborate more this documentation on what is using to verify the plugin?